src/lib/dbclient.ts

```diff
@@ -7,6 +7,8 @@ export const mongoClient = new MongoClient(url)
 
 export let db: Db
 
+export const eventsToEmitAfterDbInit: EventData[] = []
```

Comment:
Could use a better name here xd
```suggestion
export const cancelledEvents: EventData[] = []
```